### ISSUE TITLE
No 'requestduration' metrics

### ISSUE THREAD

**Issue by laullon** [vmware/wavefront-adapter-for-istio#3]
**Describe the bug**
On the `sample_operator_config.yaml` file there is 4 metrics defined:
```
    - requestsize
    - requestcount
    - requestduration
    - responsesize
```
But in WF I don't see any `requestduration` metrics.

**Comment by venilnoronha** [comment]
This is a known bug and is logged at istio/istio#8379.

**Comment by venilnoronha** [comment]
Reopening this issue as I'm waiting on a backport of the bug fix to Istio 1.0.x. Once that's done, I'll update the `istio.io/istio` package and re-publish the container and then close this.

**Comment by venilnoronha** [comment]
I've submitted a fix on Istio 1.0.x in istio/istio#9148.
